insight-hub/client.ts

```diff
@@ -21,6 +21,11 @@ const cacheKeys = {
   PROJECT_EVENT_FILTERS: "insight_hub_project_event_filters",
 }
 
+// Exclude certain event fields from the project event filters to improve agent usage
+const EXCLUDED_EVENT_FIELDS = [
+  "search" // This is searches multiple fields and is more a convenience for humans, we're removing to avoid over-matching
+];
```

Comment:
Given how this is used, is it worth using a `Set` instead of an array?
```suggestion
const EXCLUDED_EVENT_FIELDS = new Set([
  "search" // This is searches multiple fields and is more a convenience for humans, we're removing to avoid over-matching
]);
```

If so: `EXCLUDED_EVENT_FIELDS.includes` changes to `EXCLUDED_EVENT_FIELDS.has`